### PR TITLE
Made the add safe flow part of the main graph

### PR DESCRIPTION
**Pull request by jpalvarezl**
Reference  #649  .

Changes proposed in this pull request:
- The add safe load process is part of the main graph now

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/navigation/main_nav.xml [hunk 1]
```diff
@@ -6,11 +6,9 @@
     android:id="@+id/main_nav_graph.xml"
     app:startDestination="@id/safeBalancesFragment">
 
-    <include app:graph="@navigation/add_safe_nav" />
-
     <action
         android:id="@+id/action_to_add_safe_nav"
```
Comment (elgatovital) [inline app/src/main/res/navigation/main_nav.xml:10]: we don't need this global action now. You can navigate directly to addSafeFragment

File: app/src/main/res/navigation/main_nav.xml [hunk 2]
```diff
@@ -6,11 +6,9 @@
     android:id="@+id/main_nav_graph.xml"
     app:startDestination="@id/safeBalancesFragment">
 
-    <include app:graph="@navigation/add_safe_nav" />
```
Comment (elgatovital) [inline app/src/main/res/navigation/main_nav.xml:9]: add_save_nav.xml can be deleted now

Comment (jpalvarezl) [reply app/src/main/res/navigation/main_nav.xml:9]: This has been deleted already, if you look at the other file change in this PR.

File: app/src/main/res/navigation/main_nav.xml (hunk 1 above)
Comment (jpalvarezl) [reply app/src/main/res/navigation/main_nav.xml:10]: We need it for `NoSafeFragment` . If you checkout the branch and "look for usages" you will find it is still in use.

Comment (elgatovital) [reply app/src/main/res/navigation/main_nav.xml:10]: why can't you go from NoSafeFragment to AddSafeFragment?

**Pull request merged**